Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -329,13 +336,9 @@ private void OnKeyDown(object sender, KeyEventArgs e)
             }
         }
 
-        private void OnTextChanged(object sender, TextChangedEventArgs e)
+        private void MoveQueryTextToEnd()
         {
-            if (_viewModel.QueryTextCursorMovedToEnd)
-            {
-                QueryTextBox.CaretIndex = QueryTextBox.Text.Length;
-                _viewModel.QueryTextCursorMovedToEnd = false;
-            }
```

Comment:
We can add this once we want to do something else when text changed, but currently it would be better to call it when the property changed.